google/iam_helpers.go

```diff
@@ -100,6 +110,38 @@ func mergeAdditiveBindings(existing, incoming []IAMBinding) []IAMBinding {
 	return existing
 }
 
+// mergeDeleteAdditiveBindings eliminates listed members from roles in the
+// existing list.
+func mergeDeleteAdditiveBindings(existing, incoming []IAMBinding) []IAMBinding {
```

Comment:
"incoming" here is a deleted `google_*_iam_binding` resource, right?

Reply:
yes, "incoming" is the last known state prior to deletion

Reply:
I'll note that in a comment.